### PR TITLE
docs: 更新 google play beta 开发者模拟器相关文档

### PR DESCRIPTION
**Pull request by Rbqwow**
https://github.com/MaaAssistantArknights/maa-website/discussions/174#discussioncomment-10564128

### CODE REVIEW

File: docs/en-us/manual/device/windows.md [hunk 1]
```diff
@@ -72,6 +67,11 @@ const partiallySupport = shuffleArray([
         link: 'https://developer.android.com/studio/run/managing-avds',
         note: 'Theoretical support.\n\n- Starting from Android 10, Minitouch is no longer available when SELinux is in `Enforcing` mode, please switch to other touch modes, or switch SELinux **temporary** to `Permissive` mode.\n- AVD is made for debugging, it is more recommended to use other emulators designed for gaming.',
     },
+    {
+        name: 'Google Play Games (Developer)',
+        link: 'https://developer.android.com/games/playgames/emulator?hl=zh-cn',
+        note: 'Theoretical support. Hyper-V must be enabled, and a Google account must be logged in.\n\n- You need to use [custom connection](../connection.html) to connect, the ADB port is `6520`.\n- Due to the SELinux policy of Android 10 and later versions, Minitouch cannot work properly, please switch to other touch modes.\n- The first connection after each emulator startup will fail, you need to check `Attempt to kill and restart ADB process after connection failure`.',
```
Comment (Constrat) [inline docs/en-us/manual/device/windows.md:73]: Tried Play beta Dev a while back: https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues/8820
Minitouch is off limits as it requires root access anyway.
MAAtouch also doesn't seem to be working as expected (check the issue log).

So yeah, I approve the "Theoretical support"

**Review by Constrat — APPROVED** [review]

File: docs/en-us/manual/device/windows.md (hunk 1 above)
Comment (Constrat) [reply docs/en-us/manual/device/windows.md:73]: Fuck SELinux, fuck Google

Comment (Rbqwow) [reply docs/en-us/manual/device/windows.md:73]: there's a better (or best) solution called scrcpy
but it's lossy :/

Comment (Constrat) [reply docs/en-us/manual/device/windows.md:73]: Wait, maa can use the window of scrcpy as "instance"?

Comment (Rbqwow) [reply docs/en-us/manual/device/windows.md:73]: no
it's just a possible solution for MAA
scrcpy can serve as a unified solution to solve some problem like high occupancy caused by high-speed screenshots, and fix some touch issues
and it's powered by another open source community, so it has better performance and compatibility than MAA uses now
but, at all, MAA cannot use lossy images

Comment (Constrat) [reply docs/en-us/manual/device/windows.md:73]: Oh ok, as a possible solution fair enough.
MAA doesn't have to rely on its own screencap engine and just use the windows from scrcpy, I see what you mean. Also the touch is completely handled by it, unifying resolutions. MAA would just have to do template matching and ocr.

Comment (ABA2396) [reply docs/en-us/manual/device/windows.md:73]: 然而高速截图占用并不高，高的是高频ocr
image

https://github.com/user-attachments/assets/9117406e-9674-4ca0-9df4-e9c089c9cbcb

**Pull request merged**